packages/edit-site/src/components/dataviews/style.scss

```diff
@@ -38,6 +38,19 @@
 	tr {
 		border-bottom: 1px solid $gray-100;
 	}
+	.block-editor-block-preview__container {
+		height: 120px;
+		border: 1px solid $gray-300;
+		border-radius: $radius-block-ui;
+	}
+}
+
+.dataviews-grid-view {
+	.block-editor-block-preview__container {
+		height: 320px;
+		border: 1px solid $gray-300;
+		border-radius: $radius-block-ui;
+	}
```

Comment:
conditionally styling a field should be implemented in the field, not the view. That's the reason the "render" function of a field receives the "view" object as well.